Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM centos:7
+ENV JENKINS_HOME=/var/jenkins_home/
+VOLUME /var/jenkins_home/
+RUN yum -y install java
+COPY jenkins.war .
+EXPOSE 8083 8080
```

Comment:
Why do you need to mark port `8083` as exposed?
https://docs.docker.com/engine/reference/builder/#expose

Reply:
My mistake. Its need it to later project

Reply:
Now you have removed all of them. Why?